test/functorch/test_eager_transforms.py

```diff
@@ -3680,7 +3680,7 @@ def forward(self, x):
         torch.manual_seed(0)
         models = [Foo() for _ in range(num_models)]
         _, params, buffers = combine_state_for_ensemble(models)
-        result = vmap(fmodel)(params, buffers, xs)
+        result = vmap(fmodel, randomness='same')(params, buffers, xs)
```

Comment:
Why did this change?

Reply:
Forgot to undo this change. It is not needed.